Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM centos:7
+
+RUN yum update -y \
+    && yum install -y java-1.8.0-openjdk unzip wget tar which expect perl \
+    && yum clean all -y \
+    && rm -rf /var/cache/yum
+
+ARG PROJ_VERS
+ARG IMG_NAME
+
+ADD target/emissary-${PROJ_VERS}-dist.tar.gz /opt
+
+RUN ln -s /opt/emissary-${PROJ_VERS} /opt/emissary
+
+WORKDIR /opt/emissary
+
+RUN chmod 766 /opt
+RUN mkdir -p /opt/emissary-${PROJ_VERS}/localoutput/json
+RUN chmod -R a+rw /opt/emissary
+
+EXPOSE 8001
+
+
+ENTRYPOINT ["./emissary"]
+
+CMD ["server", "-a", "2", "-p", "8001"]
```

Comment:
might be useful to pass in # agents, as well other options

Reply:
I agree it might be useful to pass in agents and/or other options.  The beauty of using the Dockerfile in this manner is that these are only the default options for running this Dockerfile. For any use case where a different number of agents or other options need to be specified when doing a "docker run emissary:latest [CMD]" or similar command.

Reply:
:+1: